platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -824,7 +824,6 @@ public void setStyleUrl(@NonNull String url) {
    * @deprecated use {@link #setStyleUrl(String)} instead with versioned url methods from {@link Style}
    */
   @UiThread
-  @Deprecated
```

Comment:
That is correct. There is actually some back story behind this deprecation.. At some point in time we introduced [versioned style methods](https://github.com/mapbox/mapbox-gl-native/issues/4702). This introduced the `@Deprecated` annotation to `setStyleUrl`. Not much after that, this was reverted in https://github.com/mapbox/mapbox-gl-native/issues/5193 but the deprecated annotation was not removed. I'm definitely open to revisit the decisions made in these issues, I actually don't have a strong opinion on this,  but I think removing the annotation in this PR reflects the current `Style.java` implementation.